lib/test.gi

```diff
@@ -246,15 +251,17 @@ end;
 ##  -->
 ##  <Mark><C>showProgress</C></Mark>
 ##  <Item>If this is <K>true</K> then &GAP; prints position information
-##  and the input line before it is processed
-##  (default is <K>false</K>).</Item>
+##  and the input line before it is processed; if set to <C>"some"</C>,
+##  then GAP shows the current line of the test being processed; if set
```

Comment:
It seems that the documentation is slightly incorrect: if set to <C>"some"</C>, then GAP shows the **number of the ** current line of the test being processed, not the line itself.